README.md

```diff
@@ -16,21 +22,25 @@
 imgex is [available in Hex](https://hex.pm/packages/imgex), the package can be
 installed as:
 
-  1. Add imgex to your list of dependencies in `mix.exs`:
+Add `:imgex` to your list of dependencies in `mix.exs`:
 
-     ```elixir
-     def deps do
-       [{:imgex, "~> 0.2.0"}]
-     end
-     ```
+```elixir
+def deps do
+  [
+    {:imgex, "~> 0.2.0"}
```

Comment:
as a user I always like having a trailing comma on this so I don't have to add it after a copy-paste.
```suggestion
    {:imgex, "~> 0.2.0"},
```